page.js

```diff
@@ -0,0 +1,9 @@
+function webpage(){
```

Comment:
I think my code snippets will be a little clearer if I include a little more:

``` javascript
$("button").click(function webpage () {
   // [... your code here for what happens when the button is clicked ...]
})
```

(Don't forget to include the close parenthesis on the `click()` function!)

With the full ES6 syntax, it will look like this

``` javascript
$("button").click(() => {
   // [... your code here for what happens when the button is clicked ...]
})
```